validphys2/src/validphys/commondataparser.py

```diff
@@ -112,6 +112,7 @@ def _quick_yaml_load(filepath):
     "SHP_ASY": ("$\\eta$", "$p_T (GeV)$", "$\\sqrt{s} (GeV)$"),
     "JET_POL": ("$\\eta$", "$p_T^2 (GeV^2)$", "$\\sqrt{s} (GeV)$"),
     "DIJET_POL": ("$\\m_{1,2} (GeV)", "$\\eta_1$", "$\\eta_2$"),
+    "DY_MLL": ("$M_{ll} (GeV)$", "$M_{ll}^2 (GeV^2)$", "$\\sqrt{s} (GeV)$"),
```

Comment:
The plots produced work with this line like this^, but should it be edited to just have M_ll as a kinematic variable and remove M_ll2 and sqrts?

Reply:
You can leave it like that. That dictionary is only there due to not having all datasets implemented yet, it is necessary for the ones that have k1/k2 etc, it will be eliminated hopefully soon.